### PR TITLE
Radio and Missing Mags changes

### PR DESCRIPTION
**Pull request by Lazejun**
ACP C2 (M1911) and Vermin (Vector) were missing mags

Radio adjustment so they are the Same as in RHS Blue

### CODE REVIEW

File: A3-Antistasi/Templates/RHS_Occ_USAF_Arid.sqf [hunk 1]
```diff
@@ -120,45 +125,45 @@ groupsNATOGen = [policeOfficer,policeGrunt];
...
-vehNATOAPC = ["rhsusf_stryker_m1126_m2_wd","RHS_M2A3_BUSKIII_wd","RHS_M2A3_BUSKI_wd","rhsusf_M1237_MK19_usarmy_wd","RHS_M2A2_BUSKI_WD","rhsusf_m113_usarmy","rhsusf_m113_usarmy_MK19","rhsusf_m113_usarmy"];
-vehNATOTank = "rhsusf_m1a1fep_wd";
+vehNATOAPC = ["rhsusf_stryker_m1126_m2_d","RHS_M2A3_BUSKIII","RHS_M2A3_BUSKI","rhsusf_M1237_MK19_usarmy_d","RHS_M2A2_BUSKI_","rhsusf_m113d_usarmy","rhsusf_m113d_usarmy_MK19","rhsusf_m113_usarmy"];
+vehNATOTank = "rhsusf_m1a1fep_d";
 vehNATOAA = "RHS_M6";
 vehNATOAttack = vehNATOAPC + [vehNATOTank];
 //Boats
 vehNATOBoat = "rhsusf_mkvsoc";
-vehNATORBoat = "B_T_Boat_Transport_01_F";
+vehNATORBoat = "rhsgref_hidf_rhib";
```
Comment (Spoffy) [inline A3-Antistasi/Templates/RHS_Occ_USAF_Arid.sqf]: Right now, you're using a GREF boat in a USAF template, and I don't think we have a requirement that GREF be loaded in order to use USAF.

File: A3-Antistasi/Templates/RHS_Occ_USAF_Wdl.sqf [hunk 2]
```diff
@@ -130,11 +130,11 @@ vehNATOLight = vehNATOLightArmed + vehNATOLightUnarmed;
 //Armored
 vehNATOAPC = ["rhsusf_stryker_m1126_m2_wd","RHS_M2A3_BUSKIII_wd","RHS_M2A3_BUSKI_wd","rhsusf_M1237_MK19_usarmy_wd","RHS_M2A2_BUSKI_WD","rhsusf_m113_usarmy","rhsusf_m113_usarmy_MK19","rhsusf_m113_usarmy"];
 vehNATOTank = "rhsusf_m1a1fep_wd";
-vehNATOAA = "RHS_M6";
+vehNATOAA = "RHS_M6_wd";
 vehNATOAttack = vehNATOAPC + [vehNATOTank];
 //Boats
 vehNATOBoat = "rhsusf_mkvsoc";
-vehNATORBoat = "B_T_Boat_Transport_01_F";
+vehNATORBoat = "rhsgref_hidf_rhib";
```
Comment (Spoffy) [inline A3-Antistasi/Templates/RHS_Occ_USAF_Wdl.sqf]: Same here, we're using a GREF asset.

**Comment by SonorousMeerkat** [comment]
I don't know of anyone that uses USAF alone, _but_ if there isn't an appropriately sided one, it'd be better to leave it as vanilla.

**Comment by Lazejun** [comment]
Github is dumb, this isnt supposed to come with that PR reeeee, that Gref Boat is a Replacemebt cause USAF has no Bost so its not a Vanilla boat, which Blufor_Tropical_Boat is

**Comment by Lazejun** [comment]
The only thing i wanted Github to do was move the Edited Vanilla Blufor template, only one :(

**Comment by Spoffy** [comment]
Can you point out exactly which file you want merging @Lazejun, so I can look at only that one, and manually merge it if need be?

**Comment by Lazejun** [comment]
Vanilla_Reb_FIA_B_Altis.sqf, that was the PR  about.

The rest works and stuff but i wanted feedback before that would be put in a PR so that happend accidentally too

**Comment by Spoffy** [comment]
Merged in 783d7f19e39a816bf405338cad26429ea2beb3d5!

**Pull request closed without merging**